config/cucumber.yml

```diff
@@ -7,3 +7,4 @@ std_opts = "--format #{ENV['CUCUMBER_FORMAT'] || 'pretty'} --tags 'not @wip'"
 default: <%= std_opts %> features
 wip: --tags @wip:3 --wip features
 rerun: <%= rerun_opts %> --format rerun --out rerun.txt --strict --tags 'not @wip'
+cd: <%= std_opts %> --tags 'not @remoteapp' features
```

Comment:
Only comment, and very minor, would this be better as `servertest` or `continuousdeployment` ? `cd` is a bit ambiguous in the context of a command line

Reply:
Yeah, good point. +1 for `continuousdeployment`